src/__tests__/extensions.test.ts

```diff
@@ -23,20 +23,20 @@ describe('Extensions', () => {
   describe('all extension actions have fields labeled correctly', () => {
```

Comment:
Changes to that file should be removed

Reply:
@kubiak85 same for this file, if these changes are necessary and not required for this PR then they should go in a separate PR